### PR TITLE
Adding sketch of the manifest format documentation.

### PR DESCRIPTION
**Pull request by nyarly**

### CODE REVIEW

**Comment by sous-machine-account** [comment]
LGTM

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: doc/manifest_format.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    Env:
+      IS_CI: yes
+    # NumInstances is a guide to the number of instances that should be
+    # deployed in this cluster
+    NumInstances: 2
+    # Volumes lists the volume mappings for this deploy
+    # Generally speaking, mapping volumes breaks the stateless principle of
+    # containerized microservices and they are therefore discouraged.
+    Volumes: []
+    # Startup contains healthcheck options for this deploy.
```
Comment (samsalisbury) [inline doc/manifest_format.md]: Suggest: `# Startup contains startup healthcheck options for this deploy.`

File: doc/manifest_format.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    Startup:
+      # The path to issue healthcheck polling against.
```
Comment (samsalisbury) [inline doc/manifest_format.md]: Suggest: `# The path to issue healthcheck polling against during startup.`

File: doc/manifest_format.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+      CheckReadyURIPath: "/health"
+      # The per-check timeout.
+      CheckReadyURITimeout: 5
+      # The overall timeout before the service should be considered unhealthy.
```
Comment (samsalisbury) [inline doc/manifest_format.md]: Suggest: `# The overall time in seconds to wait for this service to become healthy.`

**Pull request merged**